src/clarity-angular/button-group/all.spec.ts

```diff
@@ -6,10 +6,12 @@
 
 import ButtonGroupSpecs from "./button-group.spec";
 import ButtonSpecs from "./button.spec";
+import LoadingButtonSpecs from "./button-loading.spec";
 import ButtonInGroupServiceSpecs from "./providers/buttonInGroup.service.spec";
 
 describe("Button Group Directives", () => {
     ButtonSpecs();
+    LoadingButtonSpecs();
```

Comment:
Same as the array of directives, this new spec should probably go in its own describe, since it's not part of button groups.